### ISSUE TITLE
Do you use the coco-text datasets for this model？

### ISSUE THREAD

**Issue by boyagesmile** [jugg1024/Text-Detection-with-FRCN#2]
Do you use the coco-text datasets for this model? or your own datasets?

**Comment by jugg1024** [comment]
yes, I use coco_text.

**Comment by franciszzj** [comment]
Are u talking about coco-text dataset divided into text and no-text to do two categories of training(classification), so that pre-training vgg network?

**Comment by jugg1024** [comment]
yes, fine-tuning vgg network using coco-text dataset, only use 'machine printed' and 'legible' text data, neg sample will be generate by frcn according to iou

**Comment by franciszzj** [comment]
@jugg1024 Thank you for your reply.
